### PR TITLE
Add OperationResponse getReturnTypeOptions

### PR DESCRIPTION
**Pull request by michaelbausor**
cc @jdpedrie @dwsupplee

### CODE REVIEW

File: src/OperationResponse.php [hunk 1]
```diff
@@ -220,6 +220,25 @@ public function getError()
...
+     * Get an array containing the values of 'operationReturnType' and/or 'metadataReturnType' if
+     * they are set. The array can be passed as the $options argument to the constructor when
+     * creating another OperationResponse object.
+     *
+     * @return array
+     */
+    public function getReturnTypeOptions()
+    {
+        $options = [];
+        if (isset($this->operationReturnType)) {
```
Comment (jdpedrie) [inline src/OperationResponse.php]: You don't need the isset checks. The properties are declared up at the top. If they're never set, they'll just be null. I'd prefer to have the keys always present with a possible null value than to have to check if the key is there or not. :)

Comment (michaelbausor) [inline src/OperationResponse.php]: Fair enough - updated to use null values instead of missing keys

File: tests/OperationResponseTest.php [hunk 2]
```diff
@@ -116,6 +124,10 @@ public function testWithOptions()
         $this->assertNull($op->getResult());
         $this->assertNull($op->getError());
         $this->assertNull($op->getMetadata());
+        $this->assertEquals([
+            'operationReturnType' => '\google\rpc\Status',
+            'metadataReturnType' => '\google\rpc\Status',
```
Comment (garrettjonesgoogle) [inline tests/OperationResponseTest.php]: Having both of these set to the same thing means your test might overlook assignment mixups in the implementation; try to have your unit test inputs use distinct values if possible.

Comment (michaelbausor) [inline tests/OperationResponseTest.php]: Fixed

**Comment by michaelbausor** [comment]
PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**